pyproject.toml

```diff
@@ -127,6 +127,7 @@ check-wheel-contents = "==0.4.0"
 GitPython = "==3.1.32"
 PyGithub = "==2.1.1"
 licensecheck = "==2024"
+pre-commit = "^3.5.0"
```

Comment:
We use `==` for dev dependencies to ensure a uniform dev environment for everyone:
```suggestion
pre-commit = "==3.5.0"
```

Reply:
Will do this.